recipes/gotranx/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gotranx" %}
+{% set name_underscore = "gotranx" %}
```

Comment:
I don't understand why you need this separate name. It seems like you could just use `name`. If that's the case, remove this and change it accordingly in `url`.

Reply:
I agree with you. I have now removed this line and changed `name_underscore` to `name` in the `url` in https://github.com/conda-forge/staged-recipes/pull/29480/commits/66cc0778f4c1aef19607795d96baf28a4c205908